app/code/Magento/ImportExport/Model/Import/Source/Zip.php

```diff
@@ -7,6 +7,8 @@
 
 /**
  * Zip import adapter.
+ *
+ * @deprecated
```

Comment:
the clear reason of depreciation should be specified , and if there is an alternative appeared , it should be specified in @ see tag